cmd/image-builder/main.go

```diff
@@ -304,27 +308,31 @@ func cmdManifestWrapper(pbar progress.ProgressBar, cmd *cobra.Command, args []st
 	}
 
 	opts := &manifestOptions{
+		ManifestgenOptions: manifestgen.Options{
```

Comment:
Frankly, I do not like Options suffix. We have one extreme when everything is `SomethingCustomizations`. I suggest to consider naming this just `Manifestgen`